sklearnex/__init__.py

```diff
@@ -50,12 +51,13 @@
     "unpatch_sklearn",
     "utils",
 ]
+onedal_iface_flag = os.getenv("OFF_ONEDAL_IFACE", "0")
```

Comment:
any particular reason os.getenv is used instead of os.environ.get?

Reply:
if this is preferred for always returning string then we can also use it in setup.py and dispatcher.py

Reply:
Both are technically same. It's just choice of usage. Sure let me change to `os.getenv` in all other places for uniformity.